src/defrag.c

```diff
@@ -1100,11 +1100,19 @@ void activeDefragCycle(void) {
 
             /* Scan the keyspace dict unless we're scanning the expire dict. */
             if (!expires_cursor)
-                cursor = dictScan(db->dict, cursor, defragScanCallback, activeDefragAlloc, db);
+                cursor = dictScanDefrag(db->dict,
+                                        cursor,
+                                        defragScanCallback,
+                                        activeDefragAlloc,
+                                        db);
```

Comment:
2 lines are sufficient. :)

Reply:
Yes... it's a matter of style. I can make it two lines though.